nstar/Cargo.toml

```diff
@@ -6,6 +6,8 @@ edition = "2018"
 
 [dependencies]
 anyhow = "1.0.34"
+clap = "^2.33"
```

Comment:
A dedicated clap dep is not needed. `structopt` reports the used `clap` version in `structopt::clap` Using a different clap version might also cause incompatibilities.